renderdoc/driver/gl/gl_debug.cpp

```diff
@@ -3725,7 +3725,8 @@ void GLReplay::InitPostVSBuffers(uint32_t eventID)
   }
 
   // get buffer data from buffer attached to feedback object
-  float *data = (float *)gl.glMapNamedBufferEXT(DebugData.feedbackBuffer, eGL_READ_ONLY);
+  float *data = (float *)gl.glMapBufferRange(eGL_TRANSFORM_FEEDBACK_BUFFER, 0,
```

Comment:
The `glMapNamedBufferEXT` should be going through the `EXT_direct_state_access` emulation in `gl_emulated.cpp` right? So isn't the real problem here isn't this function call, it's that `glMapBuffer` which the emulated function calls isn't supported on GLES?

`glMapBufferRange` is also available on desktop GL3.0 and up, so I'd rather just update the emulation to use `glMapBufferRange` instead of `glMapBuffer`.